### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by lucas-crodrigues**
Update index.html and styles.css.

Add sections:
- Works
- About Me
- Contact Me

Style sections:
- Works
- About Me
- Contact Me

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
+        <div class="aboutme1">
+            <h2 class="aboutmeh">About me</h2>
+            <p class="aboutmep">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul>
+                <li class="resume">
+                    <h3 class="resumeh">Get My Resume</h3>
+                    <a></a>
+                </li>
+            </ul>
+        </div>
```
Comment (prolajumokeoni) [inline index.html:234]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
aaa  | aaaa


- [ ] Kindly align the spacing in this section please check screenshot above ⬆️

File: index.html [hunk 2]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
+    <section class="aboutme">
+        <div class="aboutme1">
+            <h2 class="aboutmeh">About me</h2>
+            <p class="aboutmep">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul>
+                <li class="resume">
+                    <h3 class="resumeh">Get My Resume</h3>
+                    <a></a>
+                </li>
+            </ul>
```
Comment (prolajumokeoni) [inline index.html:233]: - [ ] The Get Resume should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 3]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
+                        <p class="icontextp">Ruby</p>
+                        <a></a>
+                    </li>
+                </ul>
+                <ul>
+                    <li class="seeproject1">
+                        <h3 class="seeprojecttext">See Project</h3>
+                        <a></a>
+                    </li>
+                </ul>
```
Comment (prolajumokeoni) [inline index.html:220]: - [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 4]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
```
Comment (prolajumokeoni) [inline index.html:196]: - [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 5]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
```
Comment (prolajumokeoni) [inline index.html:172]: - [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 6]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
```
Comment (prolajumokeoni) [inline index.html:148]: - [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 7]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
```
Comment (prolajumokeoni) [inline index.html:124]: - [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 8]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
```
Comment (prolajumokeoni) [inline index.html:100]: - [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 9]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
+                    <a></a>
+                </li>
+                <li class="rubybutt">
+                    <p class="icontext">Ruby</p>
+                    <a></a>
+                </li>
+            </ul>
+            <a class="seeproject">
+                <h3 class="seeprojecth">See Project</h3>
+            </a>
```
Comment (prolajumokeoni) [inline index.html:75]: - [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
uuuuuuu

File: index.html [hunk 10]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
             </li>
         </ul>
     </section>
+    <section class="works">
+        <h1 class="worksh">My Recent Works</h1>
+        <div class="divline"></div>
+        <section class="stories">
```
Comment (prolajumokeoni) [inline index.html:51]: - [ ] Kindly change the section tag to a div for each project card, this would be more semantic👍🏿

File: index.html [hunk 11]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
+                <li class="rubybutt">
+                    <p class="icontext">Ruby</p>
+                    <a></a>
+                </li>
+            </ul>
+            <a class="seeproject">
+                <h3 class="seeprojecth">See Project</h3>
+            </a>
+        </section>
+        <section class="projectssection">
```
Comment (prolajumokeoni) [inline index.html:77]: - [ ] Kindly change the section tag to a div for each project card, this would be more semantic👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @lucas-crodrigues, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 12]
```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
...
+                <li class="frameworksrubyonrails">Ruby on Rails</li>
+                <li class="frameworksRSPec">RSPec</li>
+                <li class="frameworkscapybara">CapyBara</li>
+                <li class="frameworksselenium">Selenium</li>
+            </ul>
+        </div>
+        <div class="skills">
+            <div class="skillstitle">
+                <div class="skillspoint"></div>
+                <h2 class="skillsh">Skills</h2>
```
Comment (rotimiazeez) [inline index.html:264]: - [ ] Please note that the technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in a `ul`, each of them in a `li` tag e.g
```
<ul>
<li>
<h4>language</h4>
<ul>
<li>....</li>
<li>....</li>
<li>....</li>
<li>....</li>
</ul>
</li>
<li>
<h4> framework</h4>
<ul>
<li>....</li>
<li>....</li>
<li>....</li>
<li>....</li>
</ul>
</li>
.
.
.
.
.
</ul>
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @lucas-crodrigues 👍, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @lucas-crodrigues , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lucas-crodrigues** [comment]
Oh my god, thank you so much!
Great feedback guys

**Pull request merged**